festflow/static/css/main.css

```diff
@@ -0,0 +1,65 @@
+.navbar {
+	background-color: #4285f4;
```

Comment:
Add proper comments to separate different css for different pages or even better use different css files for different views. And a common one.